recipes-bsp/u-boot/u-boot-orangepi-i96.bb

```diff
@@ -0,0 +1,35 @@
+require ${COREBASE}/meta/recipes-bsp/u-boot/u-boot.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/u-boot:"
+
+SUMMARY = "U-Boot bootloader for OrangePi i96"
```

Comment:
the file doesn't follow OE style guidelines. See u-boot-juno and u-boot-poplar that have been reformatted yesterday.